vyxal/lexer.py

```diff
@@ -100,32 +99,33 @@ def __repr__(self) -> str:
         Returns
         -------
         str
-            [name, value]
+            Token(name, value)
         """
 
-        return str([self.name.value, self.value])
+        return f"Token({repr(self.name.value)}, {repr(self.value)})"
```

Comment:
Fun fact: in a format string, you can do `f"{foo!r}"` instead of `f"{repr(foo)}"`

Doesn't really need changing, but I thought you might like to know